tox.ini

```
@@ -21,12 +21,16 @@ envlist=lint-py{36}, unittest-py{36}-dj{111}-wag{113}-slow
 #   dj20:               Use Django 2.0
 #   wag113:             Use Wagtail 1.13
 #   wag20:              Use Wagtail 2.0
+#   wag23:              Use Wagtail 2.3
+#   wag27:              Use Wagtail 2.7
 #
 # These factors are expected to combine into the follow generative environments:
 #
 #   lint-py{36}
 #   unittest-py{36}-dj{111}-wag{113}-{fast,slow}
```

It wasn't documented for `wag20`, but this should be a possible combination of factors:

```suggestion
 #   unittest-py{36}-dj{111}-wag{113,23}-{fast,slow}
```